docs/user-guide/sdh/release-hub.md

```diff
@@ -0,0 +1,265 @@
+
+1. Go to the **Rollout Release** tab.
+
+    ![Figure 16: Rollout Release Page](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/rollout-release.jpg)
+
+2. Your release needs to be marked as ready to proceed further. If it isn’t, you can mark it **Ready for release** from this screen.
+
+    ![Figure 17: Marking a Release as Ready](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/ready-release.gif)
+
+
+    Optionally, you can also do so by changing the status from **Draft** state to **Ready for release** within your release track. 
+
+    ![Figure 18: Alternative Way of Marking](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/ready-for-release.jpg)
+    
+3. Use the checkbox to select the applications belonging to the first release stage. You may use the filters on the left-hand side to make it easier.
+
+    {% embed url="https://www.youtube.com/watch?v=HPYZ4-hhoDM" caption="First Release Stage" %}
+
+4. Click **Deploy**.
+
+    If the application workflow has pre-deployment/post-deployment stage, you get a dropdown where you can specifically trigger either pre-deployment, deployment, or post-deployment stage.
+
+    ![Figure 19: Pre/Post Deployment](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/trigger-deployment.gif)
+
+5. Once the applications from the first release stage are successfully deployed, select the applications from the subsequent release stage and deploy.
+
+    {% embed url="https://www.youtube.com/watch?v=ioGZP1CCuxI" caption="Last Release Stage" %}
+
+{% hint style="warning" %}
+An application can be deployed on the tenant in the next release stage only if other applications in the previous stage are deployed successfully on the given tenant.
+{% endhint %}
+
+Here we covered the process of performing a production installation on one tenant. Similarly, you can per
+
+---
+
+## Extras
+
+### Viewing Rollout Status
+
+You can view the status of installations under `Rollout Status`. Moreover, you can go to **Rollout History** tab to view the deployment history.
+
+![Figure 20: Checking Rollout Status](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/rollout-status-history.gif)
+
+You can view the following statuses
+
+| Rollout Status    | Description                                                 |
+|-------------------|-------------------------------------------------------------|
+| All Tenants       | Overview of rollout status across all tenants installations              |
+| Yet to Trigger    | Shows tenant installations for which rollout has not yet started  |
+| Ongoing           | Shows tenant installations for which deployment is currently in progress |
+| Failed            | Shows tenant installations where rollout has encountered issues because some deployment stage failed |
+| Completed         | Shows tenant installations where deployment has successfully finished and services are live   |
```

Comment:
This needs some more clarification, but let's plan that in v2 of this doc

Reply:
Sure